include/osquery/core.h

```diff
@@ -92,10 +93,47 @@ osquery::QueryData query(const std::string& q, int& error_return, sqlite3* db);
  * virtual tables attached. This method is the main abstraction for creating
  * SQLite3 databases within osquery.
  *
+ * Note: osquery::initOsquery must be called before calling createDB in order
```

Comment:
Is there any value in iterating through the `TablePlugins` registry, looking at each `TablePlugin::columns` variable and reporting the column information **even though the table hasn't been register and may be unusable**?